lib/src/page_list_viewport_gestures.dart

```diff
@@ -442,7 +636,29 @@ class DeprecatedPanAndScaleVelocityTracker {
       return;
     }
 
+    // Check that the two swipes consequtively considered for repeated
+    // swiping acceleration are collinear
+    if (_isPossibleRepeatedAcceleratedSwipe && !((_previousLaunchVelocity.dy * velocity.dy).isNegative)) {
+      // Proceed to increase the momentum simulation initial boost to
+      // scroll faster
+      _numberOfRepeatedAcceleratedSwipes++;
+      // If the user makes tiny slow scrolls, which get deccelerated by
+      // constants due to their scale, they wont't be accelerated for the first 3 of such swipes.
+      if (_numberOfRepeatedAcceleratedSwipes > 2) {
+        _momentumSimulationInitialVelocityIncreaseScalar =
+            _repeatedSwipeVelocityIncreaseScalar(_numberOfRepeatedAcceleratedSwipes);
+        _dragIncreaseScalar = _repeatedSwipeDragIncreaseScalar(_numberOfRepeatedAcceleratedSwipes);
```

Comment:
It's not clear to me what it means to have "drag" in this name vs the other function name that doesn't have "drag". Can you try to adjust those names to clarify?